bin/container_boot

```diff
@@ -14,6 +14,9 @@ case $command in
   init)
     bundle exec rails db:migrate hyrax:default_admin_set:create hyrax:default_collection_types:create
     ;;
+  sh)
+    sh
+    ;;
```

Comment:
this was just extra so that `docker compose run web sh` can be run to bash into a shell.